### PR TITLE
4723 implement redesign of model from equation component

### PR DESCRIPTION
**Pull request by asylves1**
# Description

Layout Change to the drill down for `model-from-equation`
image

Resolves #(issue)

### CODE REVIEW

File: packages/client/hmi-client/src/components/widgets/tera-asset-block.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-	<Panel :toggleable="isToggleable">
+	<Panel :toggleable="isToggleable" :class="useDefaultStyle ? 'asset-panel' : ''">
```
Comment (blanchco) [inline packages/client/hmi-client/src/components/widgets/tera-asset-block.vue:2]: This is the vue way to assign class based on a boolean:
`:class="{ 'asset-panel': useDefaultStyle }"`

**Review by blanchco — APPROVED** [review]
okay!

File: packages/client/hmi-client/src/components/widgets/tera-slider.vue [hunk 2]
```diff
@@ -36,6 +36,10 @@ const props = defineProps({
 	tabWidth: {
 		type: String,
 		default: '50px'
+	},
+	minTabWidth: {
+		type: String,
+		default: '50px'
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/widgets/tera-slider.vue:42]: How come its 40 in the panel but 50 here?

File: packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue [hunk 3]
```diff
@@ -423,15 +426,29 @@ watch(
...
-	display: flex;
-	flex-direction: column;
-	gap: var(--gap-small);
+.no-extract-equation {
+	padding: 1rem;
+	background: #f5f5f5;
+	font-size: 12px;
+	color: gray;
+	border-radius: 3px;
+}
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:439]: use variables for paddings and border radius

Comment (shawnyama) [reply packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue:439]: and font size

File: packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue [hunk 4]
```diff
@@ -68,6 +68,7 @@
 			</tera-drilldown-header>
 			<main class="flex overflow-hidden h-full">
 				<slot name="sidebar" />
+				<slot name="content" />
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/drilldown/tera-drilldown.vue:71]: Why add this content slot? I think sidebar should do the job.

**Review by shawnyama — APPROVED** [review]

**Pull request merged**